pkg/apis/core/types.go

```diff
@@ -1442,6 +1442,12 @@ type LocalVolumeSource struct {
 	// Block devices can be represented only by VolumeMode=Block, which also requires the
```

Comment:
This comment also needs to be updated because now you can specify a block device with VolumeMode=Filesystem.

Reply:
Deleted the last three lines